chalk.c

```diff
@@ -1,11 +1,17 @@
 #include <unistd.h>
+#include <stdlib.h>  
 #include <termios.h>
+struct termios orig_termios;
```

Comment:
Keep a space between the `include` statements and variable declarations, for separating different aspects of the code